core/src/main/scala/org/apache/spark/sql/SnappyDDLParser.scala

```diff
@@ -210,6 +224,8 @@ abstract class SnappyDDLParser(session: SparkSession)
   final def SETS: Rule0 = rule { keyword(Consts.SETS) }
   final def LATERAL: Rule0 = rule { keyword(Consts.LATERAL) }
 
+  private val CURRENT_USER = "CURRENT_USER"
+
```

Comment:
This also needs support for something similar to CURRENT_GROUPS to return the groups of a user. Can be tracked for next.

Reply:
yes. that needs to be provided.